### DISCUSSION TITLE
Why do you have to build it manually?

### DISCUSSION THREAD

**Discussion by Vektor369** [alicevision/AliceVision#1765] [Q&A]
Sorry if I sound like I'm complaining. I genuinely am curious.

**Comment by natowi** [comment]
We had a build pipeline before that produced a build, however the pipeline broke and the auto build was moved to Github actions. Creating a downloadable release was not included. We "just" have to add some lines to the existing config, but someone has to do it.

**Comment by Vektor369** [comment]
The trouble is, there are so many dependencies. And then those dependencies have dependencies... I'm having an extremely difficult time even getting a few of them installed. Is it really this difficult, or am I missing something?
